skills/proxmox-admin/references/snapshot-management.md

```
@@ -0,0 +1,133 @@
+2. **Preview the rollback:**
+   ```
+   proxmox.vm.snapshot.rollback (
+     host=<PVE_HOST>,
+     vmid=<VMID>,
+     snapname="pre-upgrade-2024-01-15",
+     dry_run=true
+   )
+   ```
+
+3. **Stop the VM first** (recommended for reliable rollback):
+   ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)